### PR TITLE
Consolidate many fields into ResponseCore2

### PR DESCRIPTION
**Pull request by natebrennand**
open to renaming it to something else

### CODE REVIEW

File: common/response.go [hunk 1]
```diff
@@ -72,3 +72,12 @@ type ListResponseCore struct {
 	NextPageURI     string `json:"next_page_uri"`
 	PreviousPageURI string `json:"previous_page_uri"`
 }
+
+// TODO: rename
+type ResponseCore2 struct {
```
Comment (chelpu) [inline common/response.go]: ResourceInfo? GeneralInfo or General? Logistics? (the fields are all really general so we could name it something "general" but I h8 types/variables with numbers in them

Comment (natebrennand) [inline common/response.go]: `ResourceInfo` sgtm

Comment (chelpu) [inline common/response.go]: Aight, changing it on SIP as we speak

File: accounts/account.go [hunk 2]
```diff
@@ -62,20 +62,10 @@ func (act Account) Get(sid string) (Resource, error) {
...
-const (
-	Closed    accountStatus = "closed"
-	Suspended accountStatus = "suspended"
-	Active    accountStatus = "active"
-)
-
 // Modification represents an update to an account's info.
 type Modification struct {
 	FriendlyName string
-	Status       accountStatus
```
Comment (natebrennand) [inline accounts/account.go:78]: also worth noting that I scrapped this type

Comment (chelpu) [inline accounts/account.go:78]: Yey

**Comment by chelpu** [comment]
:thumbsup: on your part. might want to check out what I added before merging and let me know if I messed up.

**Comment by natebrennand** [comment]
lgtm, waiting for the build to pass before merging

**Comment by natebrennand** [comment]
:+1:

**Pull request merged**